<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightSettings</class>
 <widget class="QWidget" name="FlightSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>718</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Окружение</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QGroupBox" name="gbInitialSpeed">
         <property name="title">
          <string>Начальная скорость</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QLineEdit" name="leStartVelosityX">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leStartVelosityZ">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>[м/с]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Север</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Восток</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Низ</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>[м/с]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leStartVelosityY">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>[м/с]</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QGroupBox" name="gbFiles">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Файлы</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="1" colspan="2">
           <widget class="QCheckBox" name="cbSaveSim">
            <property name="text">
             <string>Сохранение симуляции в файлы</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QPushButton" name="btnSelectSaveFile">
            <property name="text">
             <string>Обзор</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QPushButton" name="btnLoadCfg">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Загрузить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Путь к файлу:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="3">
           <widget class="QLineEdit" name="leCfgFilePath">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Путь к директории:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="btnSelectCfgFile">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Обзор</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="5">
           <widget class="QCheckBox" name="cbLoadCfg">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Загрузка из конфигурационного файла</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="2" colspan="3">
           <widget class="QLineEdit" name="leSaveFilePath"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="gbWind">
         <property name="title">
          <string>Ветер</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="6">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Направление</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>[м/с]</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rbFileWind">
            <property name="text">
             <string>Из файла</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>[градусы]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbConstWind">
            <property name="text">
             <string>Постоянный</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_62">
            <property name="text">
             <string>Путь к файлу</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="5">
           <widget class="QLineEdit" name="lineEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Скорость ветра</string>
            </property>
           </widget>
          </item>
          <item row="3" column="7">
           <widget class="QPushButton" name="pushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Обзор</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="gbCoordinates">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Координаты пуска</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="4">
           <widget class="QLineEdit" name="leStartLatitude">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Угол места</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>[градусы]     </string>
            </property>
           </widget>
          </item>
          <item row="1" column="8">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>[градусы]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="leStartLongtitude">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Долгота</string>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QLineEdit" name="leStartElevation">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>[градусы]     </string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>[м]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="leStartAltitude">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLineEdit" name="leStartAzimuth">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Широта</string>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>[градусы]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Высота</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Азимут пуска</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="gbDate">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Дата и время</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>[сек]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="sbTimeStep"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Время пуска</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Шаг интегрирования</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QDateTimeEdit" name="dateTimeLaunch">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Ракета</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>1 ступень</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="WRocket" name="stage1" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>2 ступень</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="WRocket" name="stage2" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnStartSimulation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Моделирование</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WRocket</class>
   <extends>QWidget</extends>
   <header>wrocket.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
